code/modules/reagents/reagent_containers/syringes.dm

```diff
@@ -417,7 +417,34 @@
 	mode = SYRINGE_INJECT
 	update_icon()
 
+/obj/item/weapon/reagent_containers/syringe/hyronalin
+	name = "Syringe (hyronalin)"
+	desc = "Mild antiradiation medication."
+/obj/item/weapon/reagent_containers/syringe/hyronalin/New()
+	..()
+	reagents.add_reagent(HYRONALIN, 15)
+	mode = SYRINGE_INJECT
```

Comment:
that should be set in the base type definition, not in New()

add update_icon() to syringe/New() if that is necessary